src/custom/components/Header/HeaderMod.tsx

```diff
@@ -7,13 +7,9 @@ import { useTranslation } from 'react-i18next'
 
 import styled from 'styled-components'
 
-import Logo from 'assets/svg/logo.svg'
-import LogoDark from 'assets/svg/logo_white.svg'
 import { useActiveWeb3React } from 'hooks'
 import { useDarkModeManager } from 'state/user/hooks'
 import { useETHBalances /*, useAggregateUniBalance*/ } from 'state/wallet/hooks'
-// import { CardNoise } from 'components/earn/styled'
-// import { CountUp } from 'use-count-up'
```

Comment:
We can review tomorrow, but we shouldn't remove this comments. The "Mod" files are special files in the project, that are a copy of Uniswap, and they still compare line to line. This is why we keep the comments.

We try to follow this rules as much as possible: https://docs.google.com/document/d/1-4b_BXevDPq1-anyx9qFnocshlUqom5W1lYuA2Y65us/edit 

This is what has allowed us to be able to update to newer versions of uni with few conflicts

Reply:
Ah I see. Had a hunch those comments were there for a reason and not cleaned up before...

Reply:
Reverted.